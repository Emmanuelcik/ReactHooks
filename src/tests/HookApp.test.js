import {HookApp} from "../HookApp";
import {shallow} from "enzyme";
import "../setupTests";

describe("Testing <HookApp/>", () => {

    test("Should't have any errors", ()=> {
        const wrapper = shallow( <HookApp /> );
        
        expect(wrapper).toMatchSnapshot();
    })
})