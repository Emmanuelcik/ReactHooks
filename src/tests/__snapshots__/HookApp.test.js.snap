// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <HookApp/> Should't have any errors 1`] = `
<div>
  <h1>
    Hola mundo
  </h1>
</div>
`;
